tests/unit_tests/test_league_rater.py

```diff
@@ -0,0 +1,363 @@
+    assert new_score.score == 5
+
+
+def test_placement_high_rating(example_league, unplaced_player_score):
+    rating = 1500 - config.RATING_MODIFIER_FOR_PLACEMENT
+
+    new_score = LeagueRater._do_placement(example_league, unplaced_player_score, rating)
+
+    assert new_score.division_id == example_league.divisions[-1].id
+    assert new_score.game_count == unplaced_player_score.game_count + 1
+    assert new_score.score == 10
+
+
+def test_placement_low_rating(example_league, unplaced_player_score):
+    rating = -500 - config.RATING_MODIFIER_FOR_PLACEMENT
+
+    new_score = LeagueRater._do_placement(example_league, unplaced_player_score, rating)
+
+    assert new_score.division_id == example_league.divisions[0].id
+    assert new_score.game_count == unplaced_player_score.game_count + 1
+    assert new_score.score == 0
+
+
+def test_new_player(example_league):
+    current_score = LeagueScore(division_id=None, score=None, game_count=0)
+    player_rating = (380.0, 0.0)
+
+    new_score = LeagueRater.rate(
+        example_league,
+        current_score,
+        GameOutcome.VICTORY,
+        player_rating,
+    )
+
+    assert new_score.division_id == current_score.division_id
+    assert new_score.game_count == 1
+    assert new_score.score == current_score.score
+
+
+def test_placement_games(example_league):
+    current_score = LeagueScore(division_id=None, score=None, game_count=5)
+    player_rating = (380.0, 0.0)
+
+    new_score = LeagueRater.rate(
+        example_league,
+        current_score,
+        GameOutcome.VICTORY,
+        player_rating,
+    )
+
+    assert new_score.division_id == current_score.division_id
+    assert new_score.game_count == 6
+    assert new_score.score == current_score.score
+
+
+def test_promote(example_league):
+    current_score = LeagueScore(division_id=2, score=10, game_count=30)
+    player_rating = (380.0, 0.0)
+
+    new_score = LeagueRater.rate(
+        example_league,
+        current_score,
+        GameOutcome.VICTORY,
+        player_rating,
+    )
+
+    assert new_score.division_id == 3
+    assert new_score.game_count == 31
+    assert new_score.score == config.POINT_BUFFER_AFTER_DIVISION_CHANGE
+
+
+def test_demote(example_league):
+    current_score = LeagueScore(division_id=2, score=0, game_count=30)
+    player_rating = (380.0, 0.0)
+
+    new_score = LeagueRater.rate(
+        example_league,
+        current_score,
+        GameOutcome.DEFEAT,
+        player_rating,
+    )
+
+    assert new_score.division_id == 1
+    assert new_score.game_count == 31
+    assert new_score.score == 10 - config.POINT_BUFFER_AFTER_DIVISION_CHANGE
+
+
+def test_promote_in_highest_division(example_league):
+    current_score = LeagueScore(division_id=3, score=10, game_count=30)
+    player_rating = (380.0, 0.0)
+
+    new_score = LeagueRater.rate(
+        example_league,
+        current_score,
+        GameOutcome.VICTORY,
+        player_rating,
+    )
+
+    assert new_score.division_id == 3
```

Comment:
"Division has not changed"
```suggestion
    assert new_score.division_id == current_score.division_id
```